### PR TITLE
add includeoldjobs params, and fix net task display

### PR DESCRIPTION
**Pull request by orthagh**

### CODE REVIEW

File: inc/task.class.php [hunk 1]
```diff
@@ -1006,21 +1006,24 @@ function getJoblogs($task_ids = array()) {
 
          foreach ($targets as $target) {
             $item_type = key($target);
-            $item_id = current($target);
+            $item_id   = current($target);
+            $item_name = "";
+            if (strpos($item_id, '$#$')) {
```
Comment (trasher) [inline inc/task.class.php]: What if strpos() === 0?

Comment (orthagh) [inline inc/task.class.php]: good point

File: ajax/taskjob_logs.php [hunk 2]
```diff
@@ -50,13 +50,8 @@
    Session::checkCentralAccess();
 }
 
-//unlock session since access checks have been done
-session_write_close();
```
Comment (ddurieux) [inline ajax/taskjob_logs.php]: You remove that, so if have many logs, it will locks all php pages loaded

Comment (orthagh) [inline ajax/taskjob_logs.php]: ok for that, i'll move to later (to permits to write my vars)

Comment (orthagh) [inline ajax/taskjob_logs.php]: done

Comment (ddurieux) [inline ajax/taskjob_logs.php]: Ok, parfait, un petit squash et on merge ;)

**Pull request merged**